sdk/search/search-documents/review/search-documents.api.md

```diff
@@ -82,68 +81,76 @@ export interface AzureActiveDirectoryApplicationCredentials {
 export { AzureKeyCredential }
 
 // @public
-export type BlobIndexerDataToExtract = 'storageMetadata' | 'allMetadata' | 'contentAndMetadata';
+export type BlobIndexerDataToExtract = string;
```

Comment:
did these change to be extensible enums?

Reply:
Yes. I have also exposed Known values.